Manifest.toml

```diff
@@ -0,0 +1,142 @@
+# This file is machine-generated - editing it directly is not advised
```

Comment:
Accidentally checked in?

Reply:
Nope; it's nice to have a `Manifest.toml` so that sometime off in the future, people can still `pkg> instantiate` to get the exact versions that at least worked back in July of 2019.  It's kind of like a "known good point" in the space of all possible package versions.